inclusive-design/wecount.inclusivedesign.ca
# feat: rename and redirect collections (resolves #1664)

## Description

This PR makes the following changes:

- Initiatives are now at `/initiatives` and shown as Initiatives in the CMS
- Events are now at `/events` and shown as Events in the CMS
- Recount is now at `/recount` and shown as Recount in the CMS

All previous URLs are redirected.

## Steps to test

1. Visit deploy preview.
2. Explore Initiatives, Events, and Recount pages.
3. Test filtering and searching.
4. Test the CMS for these content types.
5. Test redirects from known old URLs to new URLs.

**Expected behavior:** Everything works as expected. Old URLs redirect properly.

## Additional information

Not applicable.

## Related issues

Resolves #1664.
